app/helpers/assignments_helper.rb

```diff
@@ -1,6 +1,6 @@
 module AssignmentsHelper
-	MAX = 26**4-1; # the maximum number that can be hashed to
-    CODESTRING = "ydlgknmzxjbctfiaqsrwoevuhp"; # random ordering of the alphabet
```

Comment:
I feel like 'y' is okay, and having it will increase the number of possible codes.

Reply:
I actually removed y so that that the total number of codes is divisible by 4. It was simpler than trying to significantly change the algorithm

Reply:
Oh that's interesting. Fair enough.